platform/src/main/java/org/stellar/anchor/platform/job/ReconcileFireblocksTransactionsJob.java

```diff
@@ -0,0 +1,16 @@
+package org.stellar.anchor.platform.job;
+
+import static org.stellar.anchor.util.Log.info;
+
+import org.springframework.scheduling.annotation.Scheduled;
+
+public class ReconcileFireblocksTransactionsJob {
```

Comment:
Maybe we should rename it to FireblocksTransactionsReconciliationJob to keep it consistent with cron property name?